core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -568,6 +569,10 @@ class SparkContext(config: SparkConf) extends Logging {
     _taskScheduler.postStartHook()
     _env.metricsSystem.registerSource(_dagScheduler.metricsSource)
     _env.metricsSystem.registerSource(new BlockManagerSource(_env.blockManager))
+    // Register JVMCPUSource to the metrics system if spark.metrics.cpu.time.driver.enabled==true
```

Comment:
The comment just repeats the code.

Why would you not enable this source? It seems to be always on for executors, what would be the drawback of doing the same for the driver?

Reply:
Good point. Actually, I would not mind to have this metrics gauge always on for the driver too. I don't think the overhead is high. However, I was just being cautious here following previous discussions on the implementation of this gauge for the executor instance [SPARK-25228]. I would like to ping @srowen for comment as he helped back then with [SPARK-25228].